documentation/modules/security/proc-replacing-your-own-private-keys.adoc

```diff
@@ -221,8 +221,35 @@ You can also do the same by removing the `pause-reconciliation` annotation.
 ----
 kubectl annotate Kafka _<name_of_custom_resource>_ strimzi.io/pause-reconciliation-
 ----
-
++
 On the next reconciliation, the Cluster Operator performs a rolling update of ZooKeeper, Kafka, and other components to trust the new CA certificate.
 When the rolling update is complete, the Cluster Operator will start a new one to generate new server certificates signed by the new CA key.
-
++
 If maintenance time windows are configured, the Cluster Operator will roll the pods at the first reconciliation within the next maintenance time window.
+
+.  Remove any old certificates from the secret configuration to ensure that the cluster no longer trusts the outdated certificates.
```

Comment:
I think we should make it more clear here that this should be done only after the rolling updates to move to the new CA are finished.